share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -222,8 +216,8 @@ def process_directories(self) -> Tuple[Dict, List]:
         :returns: All collections found and any errors
         """
         for directory in self._directories:
-            collection_directory = f"{directory}/ansible_collections"
-            if os.path.exists(collection_directory):
+            collection_directory = Path(directory, "ansible_collections")
```

Comment:
With the comment above, this would become:
```suggestion
            collection_directory = directory / "ansible_collections"
```